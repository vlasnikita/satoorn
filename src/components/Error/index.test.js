import ShallowRenderer from 'react-test-renderer/shallow';
import React from 'react'
import AuthErrorPage from '../Error'

it('should render auth error page correctly', () => {
    const renderer = new ShallowRenderer();
    const wrapper = renderer.render(<AuthErrorPage />)
    expect(wrapper).toMatchSnapshot()
})