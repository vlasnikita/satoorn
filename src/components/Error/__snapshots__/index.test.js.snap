// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render auth error page correctly 1`] = `
<div
  className="AuthErrorPage"
>
  <h1
    className="NotFound__header NotFound__header_main"
  >
    Что-то пошло не так во время авторизации :(
  </h1>
  <h2
    className="NotFound__header"
  >
    Нам не удалось получить данные твоего аккаунта. Попробуй авторизоваться через другую платформу  или повтори попытку позже.
  </h2>
</div>
`;
